src/smc-webapp/smc_chat.cjsx

```diff
@@ -323,27 +323,86 @@ Message = rclass
                 {cols}
             </Row>
 
+sortCopy = (arr) ->
+    return arr.slice(0).sort()
+
+are_dicts_equal = (a, b) ->
```

Comment:
I have to imagine this is in misc somewhere or use Underscore. Unless this does something special in which case that needs to be documented.